test/contracts/token_migration.aes

```diff
@@ -0,0 +1,80 @@
+payable contract TokenMigration =
```

Comment:
Maybe a reference to the original source/commit would be helpful in future. I noticed somewhere it is tracked in some repo

Reply:
I'm waiting for the contract to be merged in that repo, but yes, good point.